manual/de/Help_ch_Tools_Assistants.xml

```diff
@@ -4188,136 +4188,197 @@
+                  der Tage <emphasis>n</emphasis> an die im Voraus erinnert werden soll, wird
+                  entweder in den <guilabel>&appname; Einstellungen</guilabel> unter
+                  <xref linkend="prefs-sched" /> oder auf der Registerkarte
+                  <guilabel>Übersicht</guilabel> in dem <xref linkend="sched-editor" />
+                  angegeben.
+                </para>
+              </glossdef>
+            </glossentry>
+
+            <glossentry>
+              <glossterm>Zu Erstellen</glossterm>
+
+              <glossdef>
+                <para>Diese terminierte Buchung wird automatisch erstellt, wenn Sie <guibutton>OK</guibutton> drücken.
+                </para>
+              </glossdef>
+            </glossentry>
+
+            <glossentry>
+              <glossterm>Leer</glossterm>
+
+              <glossdef>
+                <para>Wird eine Buchung aufgelistet, für die kein Ausführungsdatum angezeigt wird, bedeutet das,
+                  dass eine terminierte Buchung im <xref linkend="sched-editor" /> angelegt ist, allerdings momentan
+                  keine Vorgänge zur Ausführung anstehen.
+                </para>
+              </glossdef>
+            </glossentry>
+          </glosslist>
+        </glossdef>
+      </glossentry>
+<!-- FIXME source: new item -->
+      <glossentry>
+        <glossterm>Wert</glossterm>
+
+        <glossdef>
+          <para>Ist für die automatische Erstellung noch die Angabe des Buchungswerts notwendig, so wird das mit
+            dem Hinweis <emphasis>(Wert benötigt)</emphasis> angezeigt. Dies kann z.B. bei einer
+            Buchung zwischen zwei Konten mit unterschiedlichen Währungen notwendig sein.
+          </para>
 
-            <varlistentry>
-              <term><guilabel>Postponed</guilabel></term>
+          <para>Sie können mit einem Doppelklick auf den Hinweistext ein Eingabefeld öffnen und einen Wert für
+            die Buchung eintragen. Das passiert ebenfalls, wenn Sie mit <guilabel>OK</guilabel> die
+            terminierte Buchung in ein Kontobuch übernehmen wollen.
+          </para>
+        </glossdef>
+      </glossentry>
+    </glosslist>
 
-              <listitem>
-                <para>If a scheduled transaction has been listed with a status <guilabel>To-create</guilabel> but you want
-                  to hold it and not have it entered to your register yet, then you can set it to
-                  <guilabel>Postponed</guilabel>. When at some later time you change it back to
-                  <guilabel>To-create</guilabel> it will be created with the original posted-date.
-                  If you want to change that you must edit the transaction after it's created.
-                </para>
-              </listitem>
-            </varlistentry>
+    <glosslist>
+      <title>Am unteren Rand des Fensters, in der Aktionsleiste, befinden sich weitere Steuerelemente:</title>
```

Comment:
Ich glaube, die Option `Erzeugte Buchungen durchsehen`gehört noch nicht zur Aktionsleiste, sondern nur die Schaltflächen.

Im Zweifelsfall noch mal in Glade gucken, wie die Bereiche heißen.